src/elements/common/annotator-context/__tests__/withAnnotations.test.tsx

```diff
@@ -42,6 +42,7 @@ describe('elements/common/annotator-context/withAnnotations', () => {
         const contextProvider = getContextProvider(wrapper);
         expect(contextProvider.exists()).toBeTruthy();
         expect((contextProvider.props() as ContextProviderProps).value).toEqual({
+            activeAnnotationId: null,
             annotation: undefined,
```

Comment:
Are these values still explicitly undefined or can they be removed here as well?